General/Operations/acknowledge.py

```diff
@@ -0,0 +1,15 @@
+# mission_ack(47) implementation
+
+def mission_acknowledge(vehicle_connection):
+    vehicle_connection.mav.mission_ack_send(
+        vehicle_connection.target_system, 
+        vehicle_connection.target_component,
+        0,0,0
+    )
+
+    msg = connection.recv_match(type='MISSION_ACK', blocking=True, timeout=5)
```

Comment:
How did you choose this timeout=5? I don't think we've implemented timeouts in our recv_match so far, but it might not be bad idea. I wonder if we should be handling timeout exceptions and sending proper information across the network so we're explicitly aware of any failures.

Reply:
@Lmah0 Because it's an acknowledgement thought it would be good to implement that after we talked about retries in the software arch meeting.

Take a look and make sure it's what you need for this use case as well.